roles/nuage_master/tasks/serviceaccount.yml

```diff
@@ -1,32 +1,19 @@
 ---
-- name: Create temporary directory for admin kubeconfig
-  command: mktemp -u /tmp/openshift-ansible-XXXXXXX.kubeconfig
-  register: nuage_tmp_conf_mktemp
-  changed_when: False
-
-- set_fact:
-    nuage_tmp_conf: "{{ nuage_tmp_conf_mktemp.stdout }}"
-
-- name: Copy Configuration to temporary conf
-  command: >
-    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{nuage_tmp_conf}}
-  changed_when: false
+- name: Load lib_openshift modules
```

Comment:
Fair point. @mtnbikenc / @ingvagabund we talked yesterday about the pattern of using meta when we can, otherwise falling back to include_role. Should we change from the include_role to finding which playbooks include a task file and adding a roles section?